packages/keystone/lib/Keystone/index.js

```diff
@@ -74,7 +74,9 @@ module.exports = class Keystone {
   createList(key, config, { isAuxList = false } = {}) {
     const { getListByKey, adapters } = this;
     const adapterName = config.adapterName || this.defaultAdapter;
-    const list = new List(key, config, {
+    const compose = fns => o => fns.reduce((acc, fn) => fn(acc), o);
+
+    const list = new List(key, compose(config.plugins || []), config, {
```

Comment:
What about running plugins over the instance of `List`/`Field` as well. We can separate the hooks of plugins, one that run on config and one that run on instance of `List/Field`

Reply:
See my other comment: https://github.com/keystonejs/keystone-5/pull/1491#discussion_r313246556